src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -44,7 +44,164 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: GET
 
+  #
+  # Single Certificate
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/applications/{applicationId}':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application.
+        schema:
+          type: string
+        in: path
+        required: true
+      - name: applicationId
+        description: The identifier of the Voter Authority Certificate to retrieve.
```

Comment:
I think this should be same as in the VCA spec:
```
      - name: applicationId
        description: The identifier of the Voter Card Application to retrieve.
```

The Certificate has a VAC number that is more like the identifier of the certificate.

Reply:
@Neil-Massey , makes sense. I will change accordingly.